### PR TITLE
[TUBEMQ-465] add new feature - copy offset from one group to another

### PR DESCRIPTION
**Pull request by EMsnap**
[TUBEMQ-465] add new feature - copy offset from one group to another

### CODE REVIEW

File: tubemq-manager/src/main/java/org/apache/tubemq/manager/controller/topic/TopicWebController.java [hunk 1]
```diff
@@ -155,11 +156,31 @@ public TubeMQResult cloneTopic(@RequestBody CloneTopicReq req) throws Exception
...
+    @PostMapping("/query/config")
     public @ResponseBody String queryTopicConfig(
         @RequestParam Map<String, String> req) throws Exception {
         String url = masterUtils.getQueryUrl(req);
         return queryMaster(url);
     }
 
 
+    /**
+     *
```
Comment (yuanboliu) [inline tubemq-manager/src/main/java/org/apache/tubemq/manager/controller/topic/TopicWebController.java:168]: add some comments here

Comment (EMsnap) [reply tubemq-manager/src/main/java/org/apache/tubemq/manager/controller/topic/TopicWebController.java]: done

**Comment by yuanboliu** [comment]
+1

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**